### ISSUE TITLE
Lightburn inverse controls

### ISSUE THREAD

**Issue by Yamilli31** [makerbase-mks/MKS-DLC32#102]
Using lasergrbl the controls are perfect using lightburn the controls are upside down, any solution?

**Comment by MKS-hosiey** [comment]
Is the motor moving in the opposite direction?

**Comment by Yamilli31** [comment]
The movement of the engines on the Mks application and on the browser are inverse to lightburns

**Comment by torenich** [comment]
Have you tried deleting and re-adding the GRBL device in Lightburn? I have a DLC32 v2 that I'm bench-testing. I had Lightburn installed and previously configured using different control boards. When I plugged the DLC32 in I got some really weird anomalies when using the arrows in the "Move" tab of Lightburn. Actually both left and right made the X motor spin the same direction!  After removing the previous GRBL device, then re-adding by letting Lightburn "Find My Laser" button, it automatically recognized the parameters of the DLC32 and everything worked after that.
